modules/cats/src/test/scala/pureconfig/module/cats/eq/package.scala

```diff
@@ -12,11 +11,33 @@ import pureconfig.module.cats.instances._
 
 package object eq {
 
-  implicit def configReaderEq[A: Eq]: Eq[ConfigReader[A]] =
-    Eq.by[ConfigReader[A], ConfigValue => ConfigReader.Result[A]](_.from)
+  implicit def configReaderEq[A: Eq](implicit arbCV: Arbitrary[ConfigValue]): Eq[ConfigReader[A]] = new Eq[ConfigReader[A]] {
+    val nSamples = 50
```

Comment:
Instead of inlining the old implementation in both methods, can you copy `catsLawsEqForFn1` as a private implicit here (with the deprecated message as a comment, for future reference)? 

Alternatively, you can also copy it as non-implicit and manually create `Eq` instances/methods for `ConfigValue => ConfigReader.Result[A]` and `A => ConfigValue`. The goal is to factor out the implementation details of `Eq` for functions and to make it clear that this was previously in cats.

Reply:
Sure. Done in 206e1c5.